website/.prettierignore

```diff
@@ -1,3 +1,4 @@
 .eslintrc.json
 tailwind.config.js
 .storybook/*
+public/mockServiceWorker.js
```

Comment:
this file will also be available in production right? at `/mockServiceWorker.js`?
intended behavior or side effect?

Reply:
According to the [mock service worker documentation](https://mswjs.io/docs/getting-started/integrate/browser) this seems to be intended (this file is auto generated by their tooling).

Reply:
https://mswjs.io/docs/getting-started/integrate/browser#start-worker

> It's not recommended to include Mock Service Worker in production. Doing so may lead to a distorted experience for your users.

I think we can leave it in right now, since we don't use a service worker at all, but we should take care of this soon, maybe in the docker image directly.